metrics/storage/fio_job.sh

```diff
@@ -206,6 +215,9 @@ function main()
 		die "Unknown runtime: $RUNTIME"
 	fi
 
+	# Drop the host page cache
+	sudo echo 3>/proc/sys/vm/drop_caches
```

Comment:
I wonder if dropping the host page cache is better done after we have created the test image rather than before - when we create the image we will populate the host cache...
Can you try this below the image creation call and see if you can see a difference?